recipe/meta.yaml

```diff
@@ -34,7 +34,7 @@ test:
     - sksparse
     - sksparse.cholmod
   commands:
-    - nosetests -sv sksparse
+    - OPENBLAS_NUM_THREADS=1 nosetests -sv sksparse
```

Comment:
I thought we couldn't really use MKL because it doesn't ship with the headers/all the dynamic libraries?